plotter/scatterColor_test.go

```diff
@@ -6,27 +6,24 @@ package plotter
 
 import (
 	"log"
-	"math/rand"
-	//"os"
 	"math"
+	"math/rand"
 	"testing"
 
 	"gonum.org/v1/plot"
 	"gonum.org/v1/plot/internal/cmpimg"
 	"gonum.org/v1/plot/palette/moreland"
 	"gonum.org/v1/plot/vg"
 	"gonum.org/v1/plot/vg/draw"
-	//"gonum.org/v1/plot/vg/vgimg"
 )
 
-// ExampleScatter_color draws some scatter points.
+// ExampleScatter_color draws a coloured scatter plot.
```

Comment:
color/colour consistency

Reply:
All public facing natural language should be written in American Standard English unless there is a good reason otherwise; `s/coloured/colored/`. Internal comments should be written in English.